cmd/swarmctl/service/list.go

```diff
@@ -31,6 +31,17 @@ var (
 				return err
 			}
 
+			nr, err := c.ListNodes(common.Context(cmd), &api.ListNodesRequest{})
+			if err != nil {
+				return err
+			}
+			readyNodes := make(map[string]struct{})
+			for _, n := range nr.Nodes {
+				if n.Status.State == api.NodeStatus_READY {
```

Comment:
@jinuxstyle: Feel free to update this PR by using `git commit --amend` and `git push -f`.